Formula/bazelisk.rb

```diff
@@ -15,11 +16,23 @@ class Bazelisk < Formula
 
   depends_on "bazel" => :build
 
+  conflicts_with "bazel", :because => "Bazelisk replaces the bazel binary"
+
+  resource "bazel_zsh_completion" do
+    url "https://raw.githubusercontent.com/bazelbuild/bazel/3.2.0/scripts/zsh_completion/_bazel"
+    sha256 "1170dc6191a08a749a4ed143323cdc60b0cd400af486eadd114ec866d513d63a"
+  end
+
   def install
     rm_f ".bazelversion" # Homebrew uses the latest bazel
     system "bazel", "build", "--config=release", "//:bazelisk-darwin"
 
     bin.install "bazel-bin/bazelisk-darwin_amd64" => "bazelisk"
+    bin.install_symlink "bazelisk" => "bazel"
```

Comment:
This is just a bad idea IMO. I see in the upstream repo that the installed binary is just `bazelisk`. Don't know if it's really needed for this software to work properly though.

Reply:
The `bazelisk` formula of Bazel's own tap also installs a symlink like this: https://github.com/bazelbuild/homebrew-tap/blob/master/Formula/bazelisk.rb#L40

The reason is that people install either `bazel` or `bazelisk`, as there's no good reason to install both - but both kind of users want their usual `bazel ...` commands to work.

In case this is not acceptable, plan B would be: Remove the symlink here, remove the zsh completions (because they also conflict with the ones that the `bazel` package installs), print a message that tells people to install the `bazel` package in addition if they wanted completions and also to install their own `bazel=bazelisk` alias or symlink if they want the old behavior.

Reply:
A workaround for this to work, could be to move the symlinking step to `post_install` method. But don't know if it is the best way to do it.

Reply:
FWIW I've just installed `bazelisk` and expected to just use `bazel`. Now I need to link it myself...